miio/interfaces/vacuum.py

```diff
@@ -0,0 +1,32 @@
+from typing import Dict, Protocol, runtime_checkable
+
+from miio import DeviceStatus
+
+FanspeedPresets = Dict[str, int]
+
+
+@runtime_checkable
+class Vacuum(Protocol):
```

Comment:
Consider rename interface to IVacuum. It is useful to understand from the name, whether it is a class or interface.